android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -76,10 +77,10 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
   public void onSaveInstanceState(Bundle outState)
   {
     super.onSaveInstanceState(outState);
-    saveEdits();
+    saveTemporaryEdits();
```

Comment:
stash я скорее пойму как “отложить, чтобы потом восстановить”. А здесь в случае сворачивания именно сохраняем изменения, чтобы потом можно было получить их обратно, без никаких восстановлений. Другой вопрос, что на карту изменения здесь не сохраняются. Оставлю комментарий наверное.
> On Mar 24, 2016, at 9:23 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/EditorFragment.java <https://github.com/mapsme/omim/pull/2506#discussion_r57364254>:
> 
> > @@ -76,10 +77,10 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
> >    public void onSaveInstanceState(Bundle outState)
> >    {
> >      super.onSaveInstanceState(outState);
> > -    saveEdits();
> > +    saveTemporaryEdits();
> Честно говоря, всё равно неудачное название. Смысл опять тот же, что и был)
> Может, stashEdits()?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2506/files/fdf2eabe22ea83dc51ce2bb2c714243bda45233d..20a4a27f1d32ada5535de3992404266c704f5df5#r57364254>